tests/bluetooth/host/cs/mocks/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+add_library(mocks STATIC
```

Comment:
Is it fine to not have license identifier there?

Reply:
added